test/config.test.js

```diff
@@ -1248,4 +1258,98 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
         assert.strictEqual(getTopLevelDomainForSameSiteResolution("http://test.com:3567"), "test.com");
         assert.strictEqual(getTopLevelDomainForSameSiteResolution("https://test.com:3567"), "test.com");
     });
+
+    it("testing no calls to the core are happening (on supertokens.init) if the cache files exist with proper values", async function () {
+        await startST();
+        await createTmpDirForTesting(); // create dir if not already present
+        await removeServerlessCache(); // if dir already present, it may still have some tmp files
+        {
+            let verifyHandshakeState = await ProcessState.getInstance().waitForEvent(
```

Comment:
Change this test to have process state in `sendRequestHelper`

Reply:
done